### PR TITLE
Home Page: Convert bottom 2 components to Rhombus on Desktop

### LINKED ISSUE #96: CSS to support converting rounded squares to rhombus
This ticket is about implementing CSS that can be reused across pages to introduce the rhombus effect for Desktop screen size:

image

In the initial implementation of the colored box areas (across all pages), we'll focus on basic layout and getting the rounded corners correct, e.g. the Mobile screen design:

image

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #96 

This is the last Rhombus ticket!

image
image
image
image
image

### CODE REVIEW

File: src/stylesheets/main.css [hunk 1]
```diff
@@ -382,6 +382,14 @@ p.important {
   .rhombus-1 {
     clip-path: polygon(0 3%, 100% 0, 96% 98%, 3% 100%);
   }
+
+  .rhombus-2 {
+    clip-path: polygon(8% 10%, 91% 0, 100% 100%, 0 100%);
```
Comment (machikoyasuda) [inline src/stylesheets/main.css:387]: Changing `91% 0` to `91% 2.5%` will better align the two top-most corners:

**Before**:

image


**After**:
image

**Review by thekaveman — COMMENTED** [review]
In tablet mode, I'm seeing a slight difference in the starting point of the two (un-rhombused) boxes -- should these be aligned at the top rather than the bottom?

image

**Comment by machikoyasuda** [comment]
@thekaveman great catch! 

have to remove `mt-2` from the 2nd div

File: src/index.html [hunk 2]
```diff
@@ -149,21 +149,21 @@ <h3 class="text-white d-block my-4">Connect with Cal-ITP</h3>
   <div class="col-md-auto">
     &nbsp;
   </div>
-  <div class="background-slate-5 shadow p-4 col-md-4 col-12 rounded rounded-4 text-center">
+  <div class="background-slate-5 shadow pt-3 pt-md-5 px-md-5 px-3 mt-2 col-md-4 col-12 text-center rhombus-3">
```
Comment (machikoyasuda) [inline src/index.html:152]:
```suggestion
  <div class="background-slate-5 shadow pt-3 pt-md-5 px-md-5 px-3 col-md-4 col-12 text-center rhombus-3">
```

**Review by thekaveman — APPROVED** [review]
Looks great!

**Pull request merged**